### PR TITLE
Feature/#111 plugin description updates

### PR DESCRIPTION
**Pull request by rasuli22**

### CODE REVIEW

File: eze/plugins/reporters/bom.py [hunk 1]
```diff
@@ -14,15 +14,15 @@ class BomReporter(ReporterMeta):
     """Python report class for echoing json dx output Bill of Materials"""
 
     REPORTER_NAME: str = "bom"
-    SHORT_DESCRIPTION: str = "json cyclonedx bill of materials reporter"
+    SHORT_DESCRIPTION: str = "Json cyclonedx bill of materials reporter"
```
Comment (Anthony-Mckale) [inline eze/plugins/reporters/bom.py:17]: JSON is the capitalisation of json, as acronym not word

File: eze/plugins/reporters/json.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ class JsonReporter(ReporterMeta):
     """Python report class for echoing all output into a json file"""
 
     REPORTER_NAME: str = "json"
-    SHORT_DESCRIPTION: str = "json output file reporter"
+    SHORT_DESCRIPTION: str = "Json output file reporter"
```
Comment (Anthony-Mckale) [inline eze/plugins/reporters/json.py:13]: same json comments

**Review by Anthony-Mckale — APPROVED** [review]
LGTM would just say are acryoins all caps ?

Json = JSON

Sarif = SARIF ?

**Review by shyam-nalluri — COMMENTED** [review]
LGTM

**Pull request merged**